builders/build-linux-rust/Dockerfile

```diff
@@ -55,4 +58,24 @@ RUN rustup component add clippy rustfmt
 RUN cargo search --limit 1
 
 # Install `sccache`
-RUN cargo install sccache --version 0.4.2
+RUN cargo install sccache --version "${SCCACHE_TAG}"
```

Comment:
While you are updating this, lets add [--locked](https://doc.rust-lang.org/cargo/commands/cargo-install.html#dealing-with-the-lockfile).

Reply:
added